### PR TITLE
[datasets][m]:- created core datasets page - refs #41

### PR DESCRIPTION
**Pull request by Mikanebu**
Create core-datasets page with introduction and full tutorial.

### CODE REVIEW

File: docs/publishers/core-datasets.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Core Datasets
+
+**Core Datasets** - important, commonly-used datasets in high quality, easy-to-use & open form.
```
Comment (anuveyatsu) [inline docs/publishers/core-datasets.md]: I'm not sure if "commonly-used" is a compound word.

File: docs/publishers/core-datasets.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+**High Quality & Reliable** - sourcing, normalizing and quality checking a set of **key reference and indicator datasets such as country codes, currencies, GDP and population**
```
Comment (anuveyatsu) [inline docs/publishers/core-datasets.md]: We've never discussed it, but should we use standard (British) English or American English? For instance, here we have "normalizing", which is spelt in American.

File: docs/publishers/core-datasets.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+<img src="https://docs.google.com/drawings/d/1Emi_N9GTv95Z_STW7XO2PVo0ykZgbgKvT30b1tpuXqI/pub?w=1136&h=318" alt="" style="min-width: 950px; margin-left: -120px;" />
+
+*Fig 1: Overview of the Curation Workflow 
+
+There are 2 areas of activity:
+
+1. Preparing datasets as Core Data Packages - finding them, cleaning them, data-packaging them
+2. Maintaining Core Data Packages - keeping them up to date with the source dataset, handling changes, responding to user queries
+
+Each of these have sub-steps which we detail below and you can contribute in any and all of these. [In fact given how many of us there are you will almost end up doing several of these at once!]
```
Comment (anuveyatsu) [inline docs/publishers/core-datasets.md]: Each of these **have** => **has**

File: docs/publishers/core-datasets.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+Often you will contribute in all 4 by taking a dataset all the way from a suggestion to a fully packaged data package published online.
+
+##### 1. Research
+
+This involves researching and selecting datasets as core datasets and adding them to the queue for packaging - no coding or data wrangling skill is needed for this
+
+* To propose a dataset for addition you [open an issue in the Registry](https://github.com/datasets/registry/issues/new) with the details of the proposed dataset.
+* Identify relevant source or sources for the dataset
+  * To propose a dataset you do not have to know where to get the data from (e.g. you could suggest “US GDP” as a core dataset without yet knowing where to get the data from)
+* Discuss with Queue Manager(s) (they will spot your submission and start commenting in the github issue)
```
Comment (anuveyatsu) [inline docs/publishers/core-datasets.md]: Very minor but we normally write "GitHub" instead of "github".

File: docs/publishers/core-datasets.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+1. [Validate](http://data.okfn.org/tools/validate) the Data Package and [review](http://data.okfn.org/tools/view) the data in the Data Package.
+2. In the review phase, you should be looking at a table with the data you have input before. That will ensure your data package is working without any issues and that it follows the same quality standards that any other package.
+3. Post a validation link and a view link in the comments for the issue in the Registry related to your Data Package.
+
+##### 4. Publishing
+
+We have a few extra specific requirements:
+
+* All Data Packages must (ultimately) be stored in a public github repo
```
Comment (anuveyatsu) [inline docs/publishers/core-datasets.md]: Same for "GitHub"

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
Small suggestions on spelling

File: docs/publishers/core-datasets.md (hunk 2 above)
Comment (Mikanebu) [reply docs/publishers/core-datasets.md]: This info is taken from http://data.okfn.org/roadmap/core-datasets. Either we can keep it or change it in their main page. Personally, I think, we can keep it as a 'normalizing'

File: docs/publishers/core-datasets.md (hunk 3 above)
Comment (Mikanebu) [reply docs/publishers/core-datasets.md]: Good point

File: docs/publishers/core-datasets.md (hunk 1 above)
Comment (Mikanebu) [reply docs/publishers/core-datasets.md]: I did some research using google Ngram Viewer, so "commonly used" would be better to use.
https://books.google.com/ngrams/graph?content=commonly+used%2Ccommonly-used&year_start=1900&year_end=2000&corpus=18&smoothing=3&share=&direct_url=t1%3B%2Ccommonly%20used%3B%2Cc0%3B.t1%3B%2Ccommonly%20-%20used%3B%2Cc0.

File: docs/publishers/core-datasets.md (hunk 4 above)
Comment (Mikanebu) [reply docs/publishers/core-datasets.md]: done

File: docs/publishers/core-datasets.md (hunk 5 above)
Comment (Mikanebu) [reply docs/publishers/core-datasets.md]: done

**Review by anuveyatsu — APPROVED** [review]

**Pull request merged**